client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationInfoBar.tsx

```diff
@@ -79,7 +79,7 @@ const InvestigationInfoBar: React.FC<Props> = ({ currentTab }: Props) => {
         const investigationInfoLogger = logger.setup('Fetching investigation Info');
         investigationInfoLogger.info('launching investigation info request', Severity.LOW);
         epidemiologyNumber !== defaultEpidemiologyNumber &&
-            axios.get(`/investigationInfo/staticInfo?investigationId=${epidemiologyNumber}`
+            axios.get(`/investigationInfo/staticInfo`
```

Comment:
you can now use ' instead of `

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1029/commits/726ef3761a2a0c382298143ba4377a2543bd0082)